docs/interfaces/notebooks.rst

```diff
@@ -212,6 +209,11 @@ could use the following configuration to launch a notebook:
    EOL
    $ det notebook start --config-file config.yaml
 
+By default, clusters that are launched by ``det deploy gcp up``, ``det deploy aws --deployment-type
+efs``, or ``det deploy aws --deployment-type fsx`` will create a Network file system that is shared
+by all the agents and automatically mounted into Notebook containers at
+``/run/determined/workdir/shared_fs/``.
+
```

Comment:
By default, launching a cluster by ``det deploy gcp up``, ``det deploy aws --deployment-type
efs``, or ``det deploy aws --deployment-type fsx`` creates a Network file system that is shared
by all the agents and is automatically mounted into Notebook containers at
``/run/determined/workdir/shared_fs/``.